src/tedi/components/layout/footer/footer.module.scss

```
@@ -0,0 +1,165 @@
+    padding: 0;
+
+    > * {
+      gap: var(--link-inner-spacing-x);
+    }
+  }
+}
+
+.tedi-footer-side {
+  display: flex;
+  flex-shrink: 0;
+  flex-direction: column;
```

Stylelint (17.12.0)

[error] Expected "flex-direction" to come before "flex-shrink" (order/properties-order)

(order/properties-order)